liota/core/package_manager.py

```diff
@@ -509,23 +534,33 @@ def _package_module_load(self, file_name, path_file_ext, file_ext):
     # This method is called to load package into current Liota process using
     # file_name (no_ext) as package identifier.
 
-    def _package_load(self, file_name, ext_forced=None, check_stack=None):
+    def _package_load(self, file_name, checksum=None, ext_forced=None, check_stack=None):
```

Comment:
I think checksum should be a mandatory argument, not optional with default as None.

I believe it is optional right now because while loading dependencies, this method is getting called, without checksum parameter. I believe this can also be a security issue. What if someone changes a dependency package and loads a trusted package which loads its modified dependency file? Please check with security team.